solver/llbsolver/mounts/mount_test.go

```diff
@@ -296,15 +296,15 @@ func TestCacheMountLockedRefs(t *testing.T) {
 	select {
 	case <-gotRef4:
 		require.FailNow(t, "mount did not lock")
-	case <-time.After(500 * time.Millisecond):
+	case <-time.After(2 * time.Second):
```

Comment:
This one does not seem to require a longer wait to make the test pass. Just makes the successful path longer.